src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -146,10 +147,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 });
             }
         } elseif ($options['multiple']) {
```

Comment:
As `expanded` is deprecated for `widget`, it doesn't make sens to use them together.